src/Upgrader/Business/PackageManagementSystem/Client/ReleaseApp/Http/Builder/HttpRequestBuilder.php

```diff
@@ -13,6 +13,9 @@
 
 class HttpRequestBuilder implements HttpRequestBuilderInterface
 {
+    /**
+     * @var array
```

Comment:
This is assoc array
```suggestion
     * @var array<string, mixed>
```
at least, maybe even string, string?

Reply:
Thanks, changed to string, string